src/query/storages/parquet/src/parquet_rs/parquet_table/table.rs

```diff
@@ -145,6 +146,17 @@ impl ParquetRSTable {
             query_kind,
             QueryKind::CopyIntoTable | QueryKind::CopyIntoLocation
         );
+        match query_kind {
+            QueryKind::CopyIntoTable | QueryKind::CopyIntoLocation => true,
+            QueryKind::Unknown => {
+                // add this branch to ensure query_kind is set
+                return Err(ErrorCode::Internal(
```

Comment:
Better to log which `query_kind` is in the error message.

Reply:
updated